crt_portal/static/js/tour.js

```diff
@@ -108,17 +108,191 @@
+      }
+    ]
+  });
+
+  savedSearchTour.addStep({
+    id: 'fourth-step',
+    text: 'Click the link to go to the Report Records table with the saved search filters applied.',
+    attachTo: {
+      element: '#name',
+      on: 'bottom'
+    },
+    classes: 'fourth-step',
+    buttons: [
+      {
+        text: 'Done',
+        action: savedSearchTour.next
+      }
+    ]
+  });
+
+  const savedSearchActionTour = new Shepherd.Tour({
+    useModalOverlay: true,
+    defaultStepOptions: {
+      cancelIcon: {
+        enabled: true
+      },
+      classes: 'usa-modal shadow-2 center',
+      scrollTo: true
+    }
+  });
+
+  savedSearchActionTour.addStep({
+    id: 'first-step',
+    text:
+      'Add a new saved search here and it will appear in the "My saved searches tab" for future reference.',
+    attachTo: {
+      element: '.intake-section-title',
+      on: 'top'
+    },
+    classes: 'first-step',
+    buttons: [
+      {
+        text: 'Next',
+        action: savedSearchActionTour.next
+      }
+    ]
+  });
+
+  savedSearchActionTour.addStep({
+    id: 'second-step',
+    text:
+      'Add a user-friendly name and description to keep track of what this query is in reference to.',
+    attachTo: {
+      element: '#id_name',
+      on: 'bottom'
+    },
+    classes: 'second-step',
+    buttons: [
+      {
+        text: 'Next',
+        action: savedSearchActionTour.next
+      }
+    ]
+  });
+
+  savedSearchActionTour.addStep({
+    id: 'third-step',
+    text: 'Add everything after the "?" in the url you want to save from the Report Records table.',
+    attachTo: {
+      element: '#id_query',
+      on: 'bottom'
+    },
+    classes: 'third-step',
+    buttons: [
+      {
+        text: 'Next',
+        action: savedSearchActionTour.next
+      }
+    ]
+  });
+
+  savedSearchActionTour.addStep({
+    id: 'fourth-step',
+    text:
+      'Check this box to make this search visible in the "Shared saved searches" tab to other portal users.',
+    attachTo: {
+      element: '#id_shared',
+      on: 'bottom'
+    },
+    classes: 'fourth-step',
+    buttons: [
+      {
+        text: 'Done',
+        action: savedSearchActionTour.next
+      }
+    ]
+  });
+
+  const tours = [
+    {
+      localStorageName: 'disposition-tour',
+      pagePath: '/form/disposition/',
+      tourName: dispositionTour
+    },
+    {
+      localStorageName: 'saved-search-tour',
+      pagePath: '/form/saved-searches/',
+      tourName: savedSearchTour
+    },
+    {
+      localStorageName: 'saved-search-action-tour',
+      pagePath: '/form/saved-searches/actions/new',
+      tourName: savedSearchActionTour
     }
+  ];
+
+  function init() {
+    const pathName = window.location.pathname;
+    tours.forEach(tour => {
+      if (localStorage.getItem(tour.localStorageName) != 'yes' && pathName == tour.pagePath) {
+        tour.tourName.start();
+        localStorage.setItem(tour.localStorageName, 'yes');
+      }
+    });
```

Comment:
Optional readability suggestion - use filter

```suggestion
    tours
    .filter(tour => {
      if (pathName != tour.pagePath) return false;
      if (localStorage.getItem(tour.localStorageName) != 'yes') return false;
      return true;
    }).forEach(tour => {
      tour.tourName.start();
      localStorage.setItem(tour.localStorageName, 'yes');
    });
```